### DISCUSSION TITLE
Github Copilot / Copilot Labs not working

### DISCUSSION THREAD

**Discussion by cblanton77** [community/community#67128] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When attempting to use the 'Code Explanation' feature in Copilot Labs in VSCode, I get the following error:

"Error generating Copilot response, please try again. Changes in the prompt or stop sequence might fix this. If this error happens repeatedly check that you are logged into GitHub and have access to both Copilot and Copilot Labs."

Also, 'Test Generation' is disabled due to this error "Test generation is currently only supported for JavaScript and TypeScript.". My code is in TypeScript so it should work.

**Comment by ubmit** [comment]
This is also happening to me. I've already tried to reinstall the extensions and to sign out and sign in again, but it's still not working.

**Comment by vishalkrishna8** [comment]
Hey @ubmit please check is there any error connecting to the GitHub API to request a token to use GitHub Copilot.

**Comment by jacksonlauder** [comment]
This is also happening to me. There is no error connecting to the GitHub API.

**Comment by hihelper** [comment]
![image](https://github.com/community/community/assets/49802184/af8d634a-c21f-4c14-b74b-4499a20152bd)

Error generating Copilot response, please try again. Changes in the prompt or stop sequence might fix this. If this error happens repeatedly check that you are logged into GitHub and have access to both Copilot and Copilot Labs
